test/hocr-simplify/hocr-simplify.tsht

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env tsht
+TESTDATA="../testdata"
+SIMPLEFILE="./tess.simple.hocr"
+
+plan 5
```

Comment:
That is the number of test cases, i.e. should be 2.

Reply:
Changed Plan 5 to Plan 3. I added two more test case, with the new char choice options.